pages/orgs/[orgId]/settings.tsx

```diff
@@ -551,6 +629,103 @@ export default function EditOrgPage() {
             {users?.length == 0 && <p className="mt-4">no data to show</p>}
             {userRole === "ADMIN" && (
               <>
+                <Heading className="text-center mt-16 mb-8">
+                  Admin Api Tokens
+                </Heading>
+                {!isOrgAdminTokenLoading && (
+                  <div>
+                    <Table
+                      className="mt-8"
+                      sx={{ minWidth: 650, maxWidth: 1000 }}
+                      aria-label="simple table"
+                    >
+                      <Thead>
+                        <Tr>
+                          <Th width="5%">
+                            <strong>Id</strong>
+                          </Th>
+                          <Th>
+                            <strong>Token</strong>
+                          </Th>
+                          <Th></Th>
+                        </Tr>
+                      </Thead>
+                      <Tbody>
+                        {orgAdminTokens?.map((token, index) => {
+                          return (
+                            <Tr key={index}>
+                              <Td>{token.id}</Td>
+                              <Td>
+                                {token.token.length > 15
```

Comment:
github does not display the token anymore after generating it and showing it once to the user
stripe only shows the api key if you press the button for it
only displaying part of the key makes it "safer" to inspect the keys (e.g. in a public setting); users also should only use the copy button and not the text field since there are less things that can go wrong (e.g. the user only copies a part of the key)

Reply:
so i will display the token as a whole in a read-only text input field (maybe with a max-width) but keep the copy button?

Reply:
Sounds good to me for now